go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/boltdb/bolt v1.3.1 // indirect
 	github.com/davecgh/go-spew v1.1.1
 	github.com/docker/distribution v2.7.1+incompatible
-	github.com/fsouza/go-dockerclient v0.0.0-20171004212419-da3951ba2e9e
+	github.com/fsouza/go-dockerclient v1.6.6-0.20200611205848-6aaf6c2d625c
```

Comment:
Need to bump this as well, otherwise `go mod tidy` fails:
```
go: finding module for package github.com/docker/libnetwork/ipamutils
go: found github.com/docker/libnetwork/ipamutils in github.com/docker/libnetwork v0.5.6
go: finding module for package github.com/Sirupsen/logrus
go: found github.com/Sirupsen/logrus in github.com/Sirupsen/logrus v1.8.0
go: github.com/openshift/origin/test/extended/images imports
	github.com/fsouza/go-dockerclient imports
	github.com/docker/docker/opts imports
	github.com/docker/libnetwork/ipamutils imports
	github.com/docker/libnetwork/osl imports
	github.com/Sirupsen/logrus: github.com/Sirupsen/logrus@v1.8.0: parsing go.mod:
	module declares its path as: github.com/sirupsen/logrus
	        but was required as: github.com/Sirupsen/logrus
```